test/unit/test_provenance.py

```diff
@@ -0,0 +1,59 @@
+import pytest
```

Comment:
Tests look great! The `rocrate` function is already covered, so good strategy to only cover the parts missing :+1: